data/human/human missions.txt

```diff
@@ -5547,3 +5547,228 @@ mission "Quicksilver Mixup 1"
+				to display
+					"Blind Man from Martini: taxi questions count" == 2
+				goto taxi
+
+			`	The taxi enters the University district proper now, and the window shows thongs of energetic university students from the University of New Sydney travelling to and from lectures and classrooms; going over steel walkways that run from building to building above you, and on footpaths beside the streets; clutching their backpacks and data pads. Your datapad shows that University isn't actually the only university district in New Sydney. The University of Chiron's campus is on the other side of town, presumably to keep the fierce competition between the two institutions from boiling over each other.`
+				to display
+					"Blind Man from Martini: taxi questions count" == 3
+				goto taxi
+
+			label hospital
+			`	Eventually, the taxi passes by a sign for the "UNS Health Sciences" quarter, which by the looks of it takes up an entire section of the University district. The taxi pulls up beside the entrance of the university hospital, where you help Douglas out with his luggage. The building is stark from the outside, to match the architecture of the surroundings, yet strangely colorful on the inside, as if the person building it decided that hospital interiors were far too unimaginative.`
+			`	After a short queue, you reach the reception with Douglas. The receptionist, placed behind a sterile-looking screen that looks far out of place with the colorful hallways, asks with a well-practiced smile, "How may I help you?"`
+			choice
+				`	"We're looking for Anita Pears. Where can we find her?"`
+			`	For a moment, the receptionist shuffles information arounds on her terminal. "You are her father, Douglas Pears?"`
+			`	"That's me," Douglas interjects before you can speak.`
+			`	She takes a few more seconds before looking at you and saying, "Ah, but we cannot let you in. Only Mr. Pears is booked in for his daughter's procedure; we have a family-only policy, to keep the privacy of our patients."`
+			choice
+				`	"All right."`
+					goto end
+				`	"Are you sure?"`
+			`	"We can't allow immediate exceptions, sorry. There's nothing I can do."`
+			label end
+			`	Douglas gives you a compassionate look, and says to the receptionist, "There'll be no worry." He turns back to you once more and says, "I'm sure that when Anita is recovered and well, we can meet again some day. I don't want to keep you from your business too long, either. I expect ship captains are busy folk." He comes up to you and, to your surprise, gives you a hug. "Thank you so much for bringing me here and coming along with me. I really can't say how I want to express my gratitude. But I hope we meet again. Farewell."`
```

Comment:
Is the thought behind the extra gratitude ("Thank you _so much_") because he's blind or because he's just that appreciative outside of the blindness? If the former, let's remove the extra thankfulness because it makes Douglas seem as though he would've been helpless without `<first>`'s help.

Reply:
I hope it's the second one (he has grown after the experience of the bombings to be more appreciative), but if it comes across like the first one I will remove it.

Reply:
Probably fine to keep it as-is, then. Just wanted to confirm!